pro/src/api/api.ts

```diff
@@ -0,0 +1,10 @@
+import { API_URL } from 'utils/config'
+
+import { APIConfiguration } from './gen'
+import { DefaultApi } from './gen/api'
```

Comment:
Petit détails: je pense qu'on peut simplifier ça à
```suggestion
import { DefaultApi } from './gen'
```